mvnvm.properties

```diff
@@ -0,0 +1 @@
+mvn_version=3.2.5
```

Comment:
What is this for? This is an old maven version, looks like this is not intentional?